webapp/src/components/Description.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Box, Typography, Link } from "@mui/material";
+import LinkIcon from "@mui/icons-material/InsertLink";
+const DOCS_URL = "https://servicenow.github.io/azimuth/user-guide";
+
+type Props = {
+  text?: string;
+  link?: string;
+};
+
+const Description: React.FC<Props> = ({ text, link }) => {
+  return (
+    <Box display="flex" gap={1} alignContent="center">
+      <Typography variant="body2">{text}</Typography>
+      {link && (
+        <Typography
+          variant="body2"
+          display="inline"
+          align="center"
+          sx={{ position: "relative", bottom: 5 }}
+        >
+          <Link href={`${DOCS_URL + link}`} color="secondary" target="_blank">
+            <LinkIcon sx={{ position: "relative", top: 5, marginRight: 0.2 }} />
+            Learn more
+          </Link>
+        </Typography>
+      )}
+    </Box>
+  );
+};
```

Comment:
I see a couple of potential improvements:
* Avoid the gap id `text` is not defined by wrapping it in a `{text && ...}`
* I think we can remove a bunch of stuff that doesn't do anything:
  - `alignContent="center"`
  - `display="inline"`
  - `align="center"`
  - `position: "relative"`
  - ``{`${DOCS_URL + link}`}`` that is the same as `{DOCS_URL + link}`
* I think we can simplify the `bottom: 5` and `top: 5` thing with a single `marginY: "-5px"`.

All together, is results in the following:
```suggestion
const Description: React.FC<Props> = ({ text, link }) => (
  <Box display="flex" gap={1}>
    {text && <Typography variant="body2">{text}</Typography>}
    {link && (
      <Link
        href={DOCS_URL + link}
        color="secondary"
        target="_blank"
        variant="body2"
      >
        <LinkIcon sx={{ marginRight: 0.5, marginY: "-5px" }} />
        Learn more
      </Link>
    )}
  </Box>
);

```

Reply:
Good question. I have seen even MUI examples with some negative margins, so I think it's completely fine.

Reply:
There are still two sub-points that are not resolved:
> * I think we can remove a bunch of stuff that doesn't do anything:
>   - `alignContent="center"`
>   - ``{`${DOCS_URL + link}`}`` that is the same as `{DOCS_URL + link}`